src/modules/link.h

```diff
@@ -21,6 +84,14 @@ struct link {
 // Public Variables
 extern struct link comm;
 
+extern uint16_t sequence_count;
```

Comment:
Shouldnt need any other public variables than those in the interface structure `struct link`

Reply:
no other public variables in link.h other than struct link